pkg/generated/openapi/def.bzl

```diff
@@ -14,17 +14,17 @@ def openapi_library(name, tags, srcs, openapi_targets=[], vendor_targets=[]):
   )
   go_genrule(
       name = "zz_generated.openapi",
-      srcs = srcs + ["//hack/boilerplate:boilerplate.go.txt"],
+      srcs = srcs + ["//" + pkg_prefix + "hack/boilerplate:boilerplate.go.txt"],
       outs = ["zz_generated.openapi.go"],
       cmd = " ".join([
         "$(location //vendor/k8s.io/code-generator/cmd/openapi-gen)",
         "--v 1",
         "--logtostderr",
-        "--go-header-file $(location //hack/boilerplate:boilerplate.go.txt)",
+        "--go-header-file $(location //" + pkg_prefix + "hack/boilerplate:boilerplate.go.txt)",
         "--output-file-base zz_generated.openapi",
-        "--output-package k8s.io/kubernetes/pkg/generated/openapi",
-        "--input-dirs " + ",".join(["k8s.io/kubernetes/" + target for target in openapi_targets] + ["k8s.io/kubernetes/vendor/" + target for target in vendor_targets]),
-        "&& cp pkg/generated/openapi/zz_generated.openapi.go $(location :zz_generated.openapi.go)",
+        "--output-package " + output_prefix + "k8s.io/kubernetes/pkg/generated/openapi",
+        "--input-dirs " + ",".join([input_prefix + target for target in openapi_targets] + ["k8s.io/kubernetes/vendor/" + target for target in vendor_targets]),
```

Comment:
this part seems weird - are we using this vendored go_genrule on the nonvendored code? I kinda feel like this should only generate openapi code for kubernetes/kubernetes, and if we need to generate for other projects, duplicate the rule there.

Reply:
The intention is to generate openapi code for both k/k standalone and k/k vendored.  Given that k/k code refers to k8s.io/pkg/generated/openapi, I presumed the best way to do this was to change the build rules to be usable in both cases.  How would you suggest generating openapi code for k/k vendored if not in this manner?

Reply:
Is it possible to use the value provided to ``go_prefix`` instead of having to define ``input_prefix`` to be the same value?